app/styles/modules/_settings.scss

```diff
@@ -0,0 +1,295 @@
+// this needs to be overridden, 
+// or done in a way that doesn't kill these everywhere
+body.settings #fox-logo,
+body.settings #about-mozilla {
+  display: none;
+}
+
+// settings header controls
+#fxa-settings-header-wrapper {
+  background: #fff;
+  border-bottom: 1px solid #b4bdc3;
```

Comment:
See https://github.com/mozilla/fxa-content-server/pull/2748/files#r34525956 (re: move this into a Sass variable)